README.md

```diff
@@ -178,4 +183,27 @@ Article.objects.all().select_json(author_name="meta__author__name")\
 
 
 
-#####For more information about raw jsonb operation, please see [PostgreSQL Documentation](http://www.postgresql.org/docs/9.4/static/functions-json.html)
+support geo search in jsonb
+===========================
+
+**require**: postgresql plugin: 
+
+1. cube
+2. earthdistance
+
+Demo
+
+```python
+Article.objects.filter(data__location__near=[39.9, 116.4,5000]) # longitude，latitude，search range
+```
+
+or 
+
+```
+Article.objects.filter(data__location__near='39.9,116.4,5000') # latitude，longitudesearch range
+```
+
+**Alert**: if you don't pass exact number of params, this filter will not be used
+
```

Comment:
Also, can you please add some documents about 
1. the format of the data in JSON,
2. some code snippet for user to install the plugin in pg, better to be in a migration operation
3. a reference link to the related PostgreSQL documents